backend/pyproject.toml

```diff
@@ -0,0 +1,106 @@
+[project]
+dynamic = ["version"]
+name = "Ibutsu Server"
+description = "A system to store and query test results and artifacts"
+authors = [
+    {name = "Raoul Snyman"},
```

Comment:
@rsnyman hey! Is there an email that you would like me to add here? I'd like to keep you as the original author here.